physics/unified_ugwp.meta

```diff
@@ -0,0 +1,1360 @@
+[ccpp-table-properties]
+  name = unified_ugwp
+  type = scheme
+  dependencies = machine.F,cires_ugwp_module.F90,ugwp_driver_v0.F,cires_ugwp_triggers.F90
```

Comment:
@climbfuji Is this OK? (dependencies defined X times?)

Reply:
Yes, additive dependencies are supported